src/node/connection_types.h

```diff
@@ -74,6 +74,12 @@ enum class ConnectionType {
      * AddrMan is empty.
      */
     ADDR_FETCH,
+
+    /**
+     * Full-rbf connections are long-lived connections used to relay transactions
+     * with peers accepting replacement without requiring replaceability signaling.
+     */
+    FULL_RBF,
```

Comment:
```
 * If adding or removing types, please update CONNECTION_TYPE_DOC in
 * src/rpc/net.cpp and src/qt/rpcconsole.cpp, as well as the descriptions in
 * src/qt/guiutil.cpp and src/bitcoin-cli.cpp::NetinfoRequestHandler. */
enum class ConnectionType {
```

If this pull moves forward would need to update these with the added connection type.

Reply:
Added the missing updates with the new connection type, including `netinfo` documentation normally.